### DISCUSSION TITLE
Follow the latest main branch updates 🤗

### DISCUSSION THREAD

**Discussion by cleardusk** [KlingAIResearch/LivePortrait#226] [Announcements]
LivePortrait’s been getting lots of updates recently, with new features and all sorts of bug fixes.

Keep up with the latest by following the main branch. 🤗

**Comment by choatehall** [comment]
I kept getting this error: ...inference.py", line 5, in <module>
    import tyro
ModuleNotFoundError: No module named 'tyro'
this happened when I ran the last step: PYTORCH_ENABLE_MPS_FALLBACK=1 python inference.py
I am completely following the instruction from here: https://github.com/KwaiVGI/LivePortrait?tab=readme-ov-file#community-resources-

it is a pity, that I failed on the last step. Otherwise this is a good one.

**Comment by cleardusk** [comment]
@choatehall Thanks for your attention. Do you try to `pip install tyro`, or just enter python env, and type `import tyro`?

**Comment by Godwinlawrence** [comment]
God bless you all

**Reply by Godwinlawrence** [reply]
Thanks
